code/modules/mob/living/carbon/human/say.dm

```diff
@@ -150,7 +150,7 @@
 				message = uppertext(message)
 				verb = "yells loudly"
 
-	if((COMIC in mutations) || (locate(/obj/item/organ/internal/cyberimp/brain/clown_voice) in internal_organs))
+	if((COMIC in mutations) || (locate(/obj/item/organ/internal/cyberimp/brain/clown_voice) in internal_organs) || istype(get_item_by_slot(slot_wear_mask), /obj/item/clothing/mask/gas/voice/clown))

```

Comment:
We really could use a trait system to avoid typecheck like this in the future

Reply:
Agreed that I don't really like how I had to implement this. Any suggestion for a cleaner way that wouldn't be too much for just this PR?